src/tacacs/bash/bash_tacplus/bash_tacplus.spec.in

```diff
@@ -0,0 +1,79 @@
+#
```

Comment:
Is this file really needed? I don't see similar requrement in other repoes, such as sonic-swss.

Reply:
We are using autoreconf tools to build this library. this file is a input file for autoreconf.

Reply:
Then, is autoreconf really needed? Is it possible to follow sonic-swss or sonic-swss-common?